appium/webdriver/extensions/action_helpers.py

```diff
@@ -149,3 +149,87 @@ def flick(self, start_x, start_y, end_x, end_y):
             .release()
         action.perform()
         return self
+
+    def zoom_element(self, element):
+        """Zooms in on an element a certain amount
+
+        :Args:
+         - element - the element to zoom
+
+        :Usage:
+            driver.zoom(element)
+        """
+        multi_touch = MultiAction()
+        size=element.size
```

Comment:
What about element.rect?
If you consider W3C or MJSONWP, you can switch the process with `self.w3c` like https://github.com/appium/python-client/blob/2cfcf090faadd302674af066b3dce130bd3ff79b/appium/webdriver/extensions/action_helpers.py#L40

Reply:
Could you follow the same format to existing docstring? Thanks.

[references]
https://github.com/appium/python-client/blob/57fafcb4c32a38d170fc8658fe017fce19d2ee44/appium/webdriver/webelement.py#L77-L88
https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html

Reply:
@KazuCocoa  According to the page:http://appium.io/docs/en/commands/interactions/actions/
and https://seleniumhq.github.io/selenium/docs/api/java/org/openqa/selenium/interactions/Actions.html#tick-org.openqa.selenium.interactions.Action-
I used the method: drag_and_drop_by_offset() 
`        if self.w3c:
            actions = ActionChains(self)
            actions\
                .drag_and_drop_by_offset(element, 0, -100)
            actions\
                .drag_and_drop_by_offset(element, 0, 100)
            actions.perform()
        else:`
but it doesn't take effect, is there anything wrong ?

Reply:
mm, could you provide me the appium log? I wonder what commands sent to Appium server.
For example, ruby lib implements the action like https://github.com/appium/ruby_lib_core/blob/master/test/functional/common_w3c_actions.rb#L21-L28 . I forgot if I tried such drag/drop method (and did not work well) tho...

Reply:
Thanks.